beetsplug/plexupdate.py

```diff
@@ -44,6 +47,14 @@ def update_plex(host, port):
     return r
 
 
+def append_token(url, token):
+    """Appends the Plex Home token to the api call if required.
+    """
+    if token:
+        url += '?X-Plex-Token={0}'.format(token)
```

Comment:
For safety, it's probably best to use:

    url += '?' + urllib.encode({'X-Plex-Token': token'})